static/scripts/rewards/render-transaction/render-ens-name.ts

```diff
@@ -1,4 +1,5 @@
 import { app } from "../app-state";
+import { Permit } from "@ubiquibot/permit-generation/types";
```

Comment:
I think these imports are not sorted properly.

Reply:
Alphabetical?